src/derivable/atom.ts

```diff
@@ -49,13 +48,19 @@ export class Atom<V> extends Derivable<V> {
      * @param newValue the new state
      */
     set(newValue: V) {
-        const oldValue = this.value;
+        const oldValue = this._value;
         if (!equals(newValue, oldValue)) {
-            this.value = newValue;
+            this._value = newValue;
             processChangedAtom(this, oldValue, this.version++);
         }
     }
 
+    /**
+     * JavaScript getter/setter access to the `#get` and `#set` methods
```

Comment:
This jsdoc seems a bit too concise. I'd quite like to explicitly see that setting `#value` calls `#set()` and getting `value` calls `#get()`.